tox.ini

```diff
@@ -4,38 +4,48 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py34, py35, py36, py37
+envlist = py27, py35, py36, py37, {py35, py36, py37}-async
```

Comment:
Why `py34` is excluded?

Reply:
Sendgrid doesn't support Python 3.4. Please see the `setup.py`:

```python
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
```

But it's a good notice because I missed Python 3.8 too